pallets/reactions/src/benchmarking.rs

```diff
@@ -78,6 +78,7 @@ benchmarks! {
         let other_kind = match reaction.kind {
             ReactionKind::Upvote => ReactionKind::Downvote,
             ReactionKind::Downvote => ReactionKind::Upvote,
+            _ => ReactionKind::Upvote,
```

Comment:
I am not sure about this. What if this is a negative emoji? Then it cannot be considered as an upvote.